Marlin/src/lcd/dwin/e3v2/dwin.cpp

```diff
@@ -2215,14 +2215,12 @@ void HMI_PauseOrStop() {
           checkkey = Back_Main;
           // Wait for planner moves to finish!
           if (HMI_flag.home_flag) planner.synchronize();
-          card.endFilePrint();
+          card.flag.abort_sd_printing = true;
+          marlin_state = MF_SD_COMPLETE;
           #ifdef ACTION_ON_CANCEL
             host_action_cancel();
           #endif
-          #ifdef EVENT_GCODE_SD_ABORT
-            Popup_Window_Home(true);
-            queue.inject_P(PSTR(EVENT_GCODE_SD_ABORT));
-          #endif
```

Comment:
With `EVENT_GCODE_SD_ABORT` removed, that means it'll skip any `EVENT_GCODE_SD_ABORT` commands a user adds in `Configuration_adv.h`.

Reply:
The `EVENT_GCODE_SD_ABORT` is applied in MarlinCore.cpp :: `abortSDPrinting`.